game.js

```diff
@@ -29,6 +31,14 @@ class Game extends Screen {
 		this.incorrectGuessChar = null
 		this.pauseUntilMilliSecond = 0 // The # of ms since the program started to pause until
 		this.puzzleRevealCountdown = 0
+    this.buttonReturnMenu = createButton('Return to menu')
```

Comment:
If the only thing the user can do from the 'win state' is return to the main menu then would it not be cleaner to just send them there after:
1. a certain amount of time has passed, or
2. they click anywhere on the screen

Rather than have an ugly button to do it?